app/facades/trivia_facade.rb

```diff
@@ -0,0 +1,9 @@
+class TriviaFacade
+  class << self
```

Comment:
no harm in using this, but from a style perspective, it might be overkill with just one method